src/components/Remark.tsx

```diff
@@ -46,23 +45,15 @@ const Remark = ({ className }: Props) => {
     return null;
   }
 
-  // TO-DO: Remove <br /> by proper margins
   return (
     <>
-      <h2>Remark</h2>
-      <Container className={className}>
-        <TextField onChange={onChange} value={remarkInput} variant="outlined" />
-        <Button variant="contained" disabled={isButtonDisabled()} onClick={sendLaneMessage}>
-          Send Remark
-        </Button>
-        <p>{estimatedFee && `Estimated source Fee: ${estimatedFee}`}</p>
-      </Container>
+      <TextField onChange={onChange} value={remarkInput} variant="outlined" fullWidth />
+      <ButtonSubmit disabled={isButtonDisabled()} onClick={sendLaneMessage}>
+        Send bridge remark from {sourceChainDetails.sourceChain} to {targetChainDetails.targetChain}
+      </ButtonSubmit>
+      <p>{estimatedFee && `Estimated source Fee: ${estimatedFee}`}</p>
```

Comment:
add <p> after the "estiamtedFee" check. There is no reason to have an empty tag if estimatedFee does not exist